.github/workflows/publish-containers.yaml

```
@@ -76,9 +82,18 @@ jobs:
           fi
 
+      - name: Build matrix arrays
+        id: build-matrix
+        run: |
+          os_json=$(echo "${SUPPORTED_OS}" | jq -R -c 'split(" ")')
+          images_json=$(echo "${SUPPORTED_IMAGES}" | jq -R -c 'split(" ")')
+          echo "os=${os_json}" >> "$GITHUB_OUTPUT"
+          echo "images=${images_json}" >> "$GITHUB_OUTPUT"
+
   publish-helm-charts-containers:
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)